src/components/ListOfWinners.js

```diff
@@ -14,8 +15,13 @@ const ListOfWinners = () => {
                src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/google/223/sports-medal_1f3c5.png" />
         </div>
         <div className="winners__container">
-          {winners.map((winner, index) => <div key={index}
-                                               className="winners__winner">{winner}</div>)}
+          {winnersList.map((winner, index) =>
+            <div key={index}
+                 className="winners__winner">
+              <div>{winner.name}</div>
```

Comment:
ja bym to zamieniła kolejnością, że najpierw avatar, nazwa a potem wynik, ale w sumie można to już przy stylowaniu poukładać :D

Reply:
taak, to na pozniej : D